web/crux-ui/src/validations/notification.ts

```diff
@@ -15,15 +15,20 @@ export const notificationSchema = yup.object().shape({
     .url()
     .when('type', (type: NotificationType, schema: yup.StringSchema<string, AnyObject, string>) => {
       let pattern: RegExp
+      let errorMsg: string
       switch (type) {
         case 'discord':
-          pattern = /^https:\/\/(discord|discordapp).com\/api\/webhooks/
+          pattern = /^https:\/\/(discord|discordapp).com\/api\/webhooks\/([\d]+)\/([a-zA-Z0-9_-]+)$/
+          errorMsg = 'https://discord(app).com/api/webhooks/ID/TOKEN'
           break
         case 'slack':
-          pattern = /^https:\/\/hooks.slack.com\/services/
+          pattern = /https:\/\/hooks.slack.com\/services\/T[0-9a-zA-Z]{10}\/B[0-9a-zA-Z]{10}\/[0-9a-zA-Z]{24}/
+          errorMsg = 'https://hooks.slack.com/services/T0000000000/B0000000000/AAAAAAAAAAAAAAAAAAAAAAAA/'
```

Comment:
Do we really want this kind of error message?
Maybe we could have a common one for all: "invalid webhook URL" or something like that

Reply:
I think it's way more informative than a simple "invalid" text. Before that we displayed the whole regex pattern, but that wasn't really user readable.

Reply:
If you find this as an example format in official Slack docs, then looks correct to me. `https://hooks.slack.com/services/T0000000000/B0000000000/AAAAAAAAAAAAAAAAAAAAAAAA/`